Namco/map.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+class Map:
+    def __init__(self):
+        """
+        Map class: initializing the map with walls and dots
+        """
+        # Loading map from the .txt file
+        self.updated_map = None
+        self.empty_map = np.loadtxt("map.txt", dtype= str, delimiter=',')
+        # Filling up the map with dots
+        self.empty_map[self.empty_map == ' '] = 'o'
```

Comment:
Esetleg 'o' helyett használhatnánk megint a '+'-t, hogy jobban elkülönüljön a 0-tól

Reply:
Jó meglátás, de a '+' helyett mondjuk egy 'x'-et javasolnék, mert így is már elég sok a vízszintes/függőleges vonal. Az átlósak szerintem jobbak lennének:
![image](https://user-images.githubusercontent.com/98428367/200192751-83943015-07ca-4e85-84b6-8635845ef60d.png)

Reply:
Wow, ez így nagyon menő :D jó ötlet!